apps/cnp/plum-frontend/aat.yaml

```diff
@@ -6,6 +6,4 @@ metadata:
 spec:
   values:
     nodejs:
-      ingressHost: plum.aat.platform.hmcts.net
-      environment:
-        RECIPE_BACKEND_URL: http://plum-recipe-backend-java
```

Comment:
Line 10, and 11 what's the requirement detail for not requiring the recipe backend anymore.